editor/src/components/canvas/canvas-strategies/relative-move-strategy.tsx

```diff
@@ -0,0 +1,144 @@
+    const selectedElements = getTargetPathsFromInteractionTarget(canvasState.interactionTarget)
+    if (selectedElements.length === 0) {
+      return false
+    }
+
+    const filteredSelectedElements = getDragTargets(selectedElements)
+    const last = filteredSelectedElements[filteredSelectedElements.length - 1]
+    const metadata = MetadataUtils.findElementByElementPath(instanceMetadata, last)
+    if (!metadata) {
+      return false
+    }
+
+    // should we also support absolute elements, for which we'll do something like `CONVERT_TO_RELATIVE`?
+    return metadata.specialSizeMeasurements.position === 'relative'
+  },
+
+  controlsToRender: [
+    {
+      control: ParentOutlines,
+      key: 'parent-outlines-control',
+      show: 'visible-only-while-active',
+    },
+    {
+      control: ParentBounds,
+      key: 'parent-bounds-control',
+      show: 'visible-only-while-active',
+    },
+  ],
+
+  fitness: (canvasState, interactionState, _sessionState) => {
+    const { interactionData, activeControl } = interactionState
+    if (!(interactionData.type === 'DRAG' && interactionData.drag != null)) {
+      return 0
+    }
+    if (activeControl.type !== 'BOUNDING_AREA') {
+      return 0
+    }
+
+    const selectedElements = getTargetPathsFromInteractionTarget(canvasState.interactionTarget)
+    if (selectedElements.length === 0) {
+      return 0
+    }
+    const filteredSelectedElements = getDragTargets(selectedElements)
+    const last = filteredSelectedElements[filteredSelectedElements.length - 1]
+    const metadata = MetadataUtils.findElementByElementPath(interactionState.latestMetadata, last)
+    if (!metadata) {
+      return 0
+    }
+    const offsets = getStyleOffsets(metadata)
+    if (!offsets) {
+      return 0
+    }
+
+    const hasOffsets =
+      offsets.left != null || offsets.top != null || offsets.right != null || offsets.bottom != null
+
+    return hasOffsets
+      ? 4 // +1 than reorder flow
+      : 1 // there should be a more structured way to define priorities (:
+  },
+
+  apply: (canvasState, interactionState, sessionState) => {
+    const { interactionData } = interactionState
+    if (!(interactionData.type === 'DRAG' && interactionData.drag != null)) {
+      return emptyStrategyApplicationResult
+    }
+
+    const selectedElements = getTargetPathsFromInteractionTarget(canvasState.interactionTarget)
+    if (selectedElements.length === 0) {
+      return emptyStrategyApplicationResult
+    }
+
+    const filteredSelectedElements = getDragTargets(selectedElements)
+    const last = filteredSelectedElements[filteredSelectedElements.length - 1]
+    const metadata = MetadataUtils.findElementByElementPath(sessionState.startingMetadata, last)
+    if (metadata == null) {
+      return emptyStrategyApplicationResult
+    }
+    const offsets = getStyleOffsets(metadata)
+    if (!offsets) {
+      return emptyStrategyApplicationResult
+    }
```

Comment:
It seems that with the use of `applyMoveCommon` this code is now effectively just duplicating the `isApplicable` or `fitness` functions, in which case we should probably just directly reuse those at the start of this function (we're already following that pattern in a bunch of other strategies)

Reply:
https://github.com/concrete-utopia/utopia/pull/2604#:~:text=98067a4-,reuse%20fitting%20logic%20when%20applying,-b635766